packages/integration-sdk-runtime/src/execution/dependencyGraph.ts

```diff
@@ -345,7 +351,14 @@ export function executeStepDependencyGraph<
         }
 
         if (status !== StepResultStatus.CACHED) {
-          await step.executionHandler(context);
+          if (wrapper) {
```

Comment:
I would consider setting a default value for `stepWrapper` in order to avoid needing an `if/else` statement here:

```diff
export function executeStepDependencyGraph<
  TExecutionContext extends ExecutionContext,
  TStepExecutionContext extends StepExecutionContext,
>({
  executionContext,
  inputGraph,
  stepStartStates,
  duplicateKeyTracker,
  graphObjectStore,
  dataStore,
  createStepGraphObjectDataUploader,
  beforeAddEntity,
  beforeAddRelationship,
  afterAddEntity,
  afterAddRelationship,
- stepWrapper,
+ stepWrapper = (_, executionHandler) => executionHandler(),
}: {
  executionContext: TExecutionContext;
  inputGraph: DepGraph<Step<TStepExecutionContext>>;
  stepStartStates: StepStartStates;
  duplicateKeyTracker: DuplicateKeyTracker;
  graphObjectStore: GraphObjectStore;
  dataStore: MemoryDataStore;
  createStepGraphObjectDataUploader?: CreateStepGraphObjectDataUploaderFunction;
  beforeAddEntity?: BeforeAddEntityHookFunction<TExecutionContext>;
  beforeAddRelationship?: BeforeAddRelationshipHookFunction<TExecutionContext>;
  afterAddEntity?: AfterAddEntityHookFunction<TExecutionContext>;
  afterAddRelationship?: AfterAddRelationshipHookFunction<TExecutionContext>;
  stepWrapper?: StepWrapperFunction<TStepExecutionContext>;
}): Promise<IntegrationStepResult[]> {

  // ...

        if (status !== StepResultStatus.CACHED) {
+         await stepWrapper({ step }, () => step.executionHandler());
-         if (wrapper) {
-           await wrapper(
-             step,
-             async () => await step.executionHandler(context),
-           );
-         } else {
-           await step.executionHandler(context);
-         }